packages/core/src/fragments/IfcGeometryTiler/src/base-types.ts

```diff
@@ -1,30 +1,39 @@
-/**
- * A dictionary of geometries streamed from a server. Each geometry is identified by a unique number (id), and contains information about its bounding box, whether it has holes, and an optional file path for the geometry data.
- */
+interface GeometrySettings {
+  /** The bounding box of the geometry as a Float32Array. */
+  boundingBox: Float32Array;
+  /** A boolean indicating whether the geometry has holes. */
+  hasHoles: boolean;
+}
+
 export interface StreamedGeometries {
-  [id: number]: {
-    /** The bounding box of the geometry as a Float32Array. */
-    boundingBox: Float32Array;
-    /** A boolean indicating whether the geometry has holes. */
-    hasHoles: boolean;
-    /** An optional file path for the geometry data. */
-    geometryFile?: string;
-  };
+  [id: number]: GeometrySettings;
 }
 
 /**
  * A streamed asset, which consists of multiple geometries. Each geometry in the asset is identified by a unique number (geometryID), and contains information about its transformation and color.
  */
 export interface StreamedAsset {
-  /** The unique identifier of the asset. */
   id: number;
   /** An array of geometries associated with the asset. */
-  geometries: {
-    /** The unique identifier of the geometry. */
+  geometries: Array<{
     geometryID: number;
-    /** The transformation matrix of the geometry as a number array. */
+    /** The transformation matrix of the geometry. */
     transformation: number[];
-    /** The color of the geometry as a number array. */
     color: number[];
-  }[];
+  }>;
+}
+
+export type GeometryTileFileName = `${string}-${number}`;
+export type GlobalDataFileName = `${string}-global`;
```

Comment:
the fact that these files have to use a specific suffix is not clear without reading the docs. by using string literal, we can force the consumers to use a specific naming.

the only other reasonable option I see would be to make the names explicit during configuration.